src/Middleware/TrustCloudflare.php

```diff
@@ -52,7 +52,7 @@ public function handle(Request $request, Closure $next)
         // it has a consistent format containing only one IP.
         $request->headers->set('X-Forwarded-For', $request->header('CF-Connecting-IP'));
 
-        Request::setTrustedProxies($this->proxies, Request::HEADER_X_FORWARDED_ALL);
+        Request::setTrustedProxies($this->proxies, Request::HEADER_X_FORWARDED_FOR | Request::HEADER_X_FORWARDED_HOST | Request::HEADER_X_FORWARDED_PORT | Request::HEADER_X_FORWARDED_PROTO | Request::HEADER_X_FORWARDED_AWS_ELB);
```

Comment:
We don't need the `Request::HEADER_X_FORWARDED_AWS_ELB` header as it's for Cloudflare